code/modules/admin/admin_ranks.dm

```diff
@@ -171,21 +171,19 @@ GLOBAL_PROTECT(admin_ranks)
 	#endif
 
 
-/proc/load_admins(target = null)
-	if(IsAdminAdvancedProcCall())
-		to_chat(usr, "<span class='admin prefix'>Admin Reload blocked: Advanced ProcCall detected.</span>")
-		return

```

Comment:
This was removed because there is literally a verb given to all admins that calls this proc.